packages/members-api/lib/repositories/product.js

```diff
@@ -278,7 +281,7 @@ class ProductRepository {
             id: data.id || options.id
         });
 
-        if (this._stripeAPIService.configured) {
+        if (this._stripeAPIService.configured && data.type !== 'free') {
```

Comment:
Can we read this from `product` instead, and then not need it to be passed into the method with `data`?

Imagine a case where you pass in data like: `{name: 'whatever'}` - then `data.type !== 'free'` will be true - and this block will execute

Reply:
🤔  Yeah true, i have updated this piece of code now to anyways first get product details before editing it, as we also need it to make sure we don't update the `name` for free tier

Reply:
Updated